calendar-helper/index.js

```diff
@@ -23,7 +23,12 @@ class CalendarHelper {
         minDate: null,
         maxDate: null,
         ignoreWeekDays: [],
-        shortcutButtons: [],
+        shortcutButtons: {
+          buttons: [],
+          placement: 'top',
+          callbackSuffix: '',
+          shouldPrefixButtonCallbacks: false,
+        },
         hideIgnoredWeeks: false,
       },
       options
```

Comment:
This is not handling the case of input `shortcutButtons` as array. We need to support this case for backwards compatibility. You can apply this patch to fix:
```suggestion
        hideIgnoredWeeks: false,
      },
      options
    );

    const inputButtons = Array.isArray(options.shortcutButtons)
      ? { buttons: options.shortcutButtons }
      : options.shortcutButtons;

    this.options.shortcutButtons = Object.assign(
      {
        buttons: [],
        placement: 'top',
        callbackSuffix: '',
        shouldPrefixButtonCallbacks: false,
      },
      inputButtons
```